src/uwtools/drivers/chgres_cube.py

```diff
@@ -2,45 +2,21 @@
 A driver for chgres_cube.
 """
 
-from datetime import datetime
 from pathlib import Path
-from typing import Optional
 
 from iotaa import asset, task, tasks
 
 from uwtools.config.formats.nml import NMLConfig
-from uwtools.drivers.driver import Driver
+from uwtools.drivers.driver import DriverCycleBased
 from uwtools.strings import STR
 from uwtools.utils.tasks import file
 
 
-class ChgresCube(Driver):
+class ChgresCube(DriverCycleBased):
```

Comment:
This is the first example of a theme that repeats below (beyond the `driver.py` diff): Simplify the driver class by inheriting from a new, more-specific base class that defines some common behavior. Some imports are no longer needed, and the constructor and `_taskname()` method from the immediate parent class is correct for this class.